resources/lang/en/general.php

```diff
@@ -1,7 +1,32 @@
 <?php
 
 return [
-    'all'        => 'All',
-    'date'       => 'Date',
-    'empty-logs' => 'The list of logs is empty!',
+    'all'             => 'All',
+    'date'            => 'Date',
+    'empty-logs'      => 'The list of logs is empty!',
+    'dashboard'       => 'Dashboard',
+    'entries'         => 'entry|entries',
+    'logs'            => 'Logs',
+    'log'             => 'log',
+    'log_info'        => 'log informations',
+    'download'        => 'Download',
+    'delete'          => 'Delete',
+    'file_path'       => 'File Path :',
+    'log_entries'     => 'Log Entries :',
+    'size'            => 'Size :',
+    'created_at'      => 'Created At :',
+    'updated_at'      => 'Updated At :',,
```

Comment:
Can you remove the duplicated `,` ?

Check: https://travis-ci.org/ARCANEDEV/LogViewer/jobs/260658666#L394